oaknational/Oak-Web-Application
# chore: about us a11y heading tags [LESQ-940]

## Description

Music year: 1975

[Ticket](https://www.notion.so/oaknationalacademy/Change-headers-on-about-us-board-page-7261d1ef27464d4b9e8fcf0cd2ef13d5?pvs=4)
- Swap out heading for p tag on document headings on the about us/board page


## How to test

1. Go to https://deploy-preview-2594--oak-web-application.netlify.thenational.academy/about-us/board
2. Inspect the documents cards
3. You should see a regular p tag for the text not a header
